### PR TITLE
Add celery tasks to update db

### PR DESCRIPTION
**Pull request by epedzich**

### CODE REVIEW

File: cinemas_repertoire/tasks.py [hunk 1]
```diff
@@ -1,7 +1,91 @@
...
+        dates = get_dates(cinema_id=cinema['id'])
+        for date in dates:
+            update_events_per_cinema.delay(cinema['id'], date)
+            update_movies_per_cinema.delay(cinema['id'], date)
+
+
+app.conf.beat_schedule = {
+    'update_from_api_once_a_day': {
+        'task': 'cinemas_repertoire.tasks.update_cinemas_events_and_movies',
+        'schedule': crontab(minute=0, hour=0)
```
Comment (lukasz-michalak) [inline cinemas_repertoire/tasks.py]: I would make schedule configurable via django settings

File: cinemas_repertoire/tasks.py [hunk 2]
```diff
@@ -1,7 +1,91 @@
...
+                                                      many=True)
+    for data in serializer.data:
+        kwargs = data.copy()
+        film, _ = CinemaCityMovie.objects.update_or_create(cc_movie_id=kwargs.pop('cc_movie_id'), defaults=kwargs)
+
+
+@app.task()
+def update_cinemas_events_and_movies():
+    update_cinemas.delay()
+    for cinema in get_cinemas().values():
```
Comment (lukasz-michalak) [inline cinemas_repertoire/tasks.py]: there is no guarantee that previous `update_cinemas.delay()` will be done/ready at this point. so it is quite possible you will be updating yesterdays data.

**Review by lukasz-michalak — CHANGES_REQUESTED** [review]

File: cinemas_repertoire/tasks.py (hunk 1 above)
Comment (epedzich) [reply cinemas_repertoire/tasks.py]: Somethng like: <NAME_OF_TASK>_CRON = crontab(minute=0, hour=0)?

Comment (lukasz-michalak) [reply cinemas_repertoire/tasks.py]: tak, coś tego typu

**Review by lukasz-michalak — APPROVED** [review]

**Pull request merged**